### DISCUSSION TITLE
Soooo... how do I turn this thing off?

### DISCUSSION THREAD

**Discussion by melak** [community/community#56520] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's all in the title.

**Comment by Serena867** [comment]
I would also prefer to turn this feature off entirely

**Comment by dawnmist** [comment]
I'd also like to turn the new "feed" off entirely.

Why on earth would I want to know about someone starring a repository that I've never used, never contributed to, don't know any of the developers, and have absolutely no interest in? It'd potentially be useful if the results were at least filtered to repositories you had been involved with, but displaying updates for random repos makes it _completely irrelevant noise_ that simply means I have to go through extra steps to see the repos that I am interested in (e.g. my own repos!).

**Comment by baryluk** [comment]
100% of what I got there is irrelevant and just noise. I never commented, subscribed, contributed or visited any of these repos it is showing me.

Why would anybody care that some random person X starred random project Y?

How possibly this could be useful to anybody?

Previous notifications where not perfect, but at least all of them were relevant to me.

**Comment by IdrisQe** [comment]
I'd much prefer the way it was before back. This is just cluttered and hard to read, even if manually disabling certain parts.

**Comment by wwahammy** [comment]
This feature is infuriating. Quit trying to turn Github into Facebook and just leave us alone.

**Comment by SaswatPadhi** [comment]
Same here. Since it's in "beta" stage, shouldn't there be a way for users to opt out?!
I don't want my GitHub feed to look like my Facebook feed.

**Comment by a3s7p** [comment]
it sucks

**Comment by victor-david** [comment]
Here's a novel idea for you. Let users decide what they'd like to see. I don't want to see a bunch of clutter and noise from other repos that I don't care about. Maybe consider NOT deciding for us what you think we'd like to see. We're adults here. We can make up our own minds. This is is just more useless junk.

**Comment by diosmosis** [comment]
Literally nothing in the feed is something I actually want to see...

**Reply by diosmosis** [reply]
Unchecking all the filters empties it for me.
